### PR TITLE
Column Alignment and Examples

### LINKED ISSUE #133: Allow center align table cells
Add class for `<td>` to `text-align: center;`. 

For short list values, where centering content reads better, such as shown in the 2nd and 3rd columns below. 

Screen Shot 2023-06-30 at 8 42 58 AM

### PR DESCRIPTION
**Pull request by SteveLLamb**
Closes #133 

Added samples to new annex

### CODE REVIEW

File: css/smpte.css [hunk 1]
```diff
@@ -288,6 +288,42 @@ tbody {
   text-align: left;
 }
 
+td.center-cell {
+  text-align: center;
+}
+
+table.col-1-center td:nth-child(1) {
```
Comment (palemieux) [inline css/smpte.css:295]: Shouldn't this be `tr td:nth-child(n)` as in nth td child of a tr?

**Review by palemieux — CHANGES_REQUESTED** [review]

File: css/smpte.css (hunk 1 above)
Comment (SteveLLamb) [reply css/smpte.css]: Apparently not, from what I'm reading it's always the 2nd `td` in each `tr` regardless. It appears to have the same effect either way. But I"ll change this as it make more sense to me as well that way.

File: doc/main.html [hunk 2]
```diff
@@ -1230,6 +1235,564 @@ <h3>Git commit hooks</h3>
...
+              <td>Name 03</td>
+              <td><code>Code 3</code></td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+      <div class="example">
+        <pre>
+&lt;table id="tab-sample-tabledata-rowspan"&gt;
```
[the remark this answers is not in the document]
Comment (SteveLLamb) [reply doc/main.html]: Addressed in https://github.com/SMPTE/html-pub/pull/135/commits/3ebfe8008d1fcda9c4376ea15b61704faf265698

File: css/smpte.css (hunk 1 above)
Comment (SteveLLamb) [reply css/smpte.css]: Addressed in https://github.com/SMPTE/html-pub/pull/135/commits/3ebfe8008d1fcda9c4376ea15b61704faf265698

File: doc/main.html [hunk 3]
```diff
@@ -1230,6 +1235,330 @@ <h3>Git commit hooks</h3>
...
+              <td><code>Code 3</code></td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p class="note">
+          Classes are available to center align columns 1-8 (as <code>col-x-center</code>), and mutliple classes to <code>table</code> may be added to align more columns as needed.
+        </p>
+        <p class="note">
+          For tables over 8 columns wide, it is advised to consider reformatting for better readability.
```
Comment (palemieux) [inline doc/main.html:1319]:
```suggestion
          Tables over 8 columns wide should not be used and instead split into multiple smaller tables.
```

**Review by palemieux — CHANGES_REQUESTED** [review]
This is awesome

File: doc/main.html [hunk 4]
```diff
@@ -1230,6 +1235,330 @@ <h3>Git commit hooks</h3>
...
 
+  <section class="annex" id="sec-table-examples">
+    <h2>Table Examples</h2>
+    <section id="sec-table-alignment">
+      <h3>Alignment</h3>
+      <p>
+        By default, table headers within <code>thead</code> are set to <code>text-align: center;</code>, and cells within the body <code>tbody</code> are set to <code>text-align: left;</code>, as the below example shows.
+      </p>
+
+      <div class="example">
```
[the remark this answers is not in the document]
Comment (SteveLLamb) [reply doc/main.html]: Update via https://github.com/SMPTE/html-pub/pull/135/commits/6007a5b4357edfadbcd492bf78f2ce7bc5ba5e94

**Review by palemieux — APPROVED** [review]

**Pull request merged**